transports/notifications/index.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * index.js
+ * Module for transferring messages through desktop notifications
+ */
+'use strict';
+
+/**
+ * Importing modules
+ */
+const io = require('../../includes/io.js'),
+	util = require('../../includes/util.js'),
+	Transport = require('../transport.js'),
+	notifier = require('node-notifier'),
+	opn = require('opn');
+var config,
```

Comment:
`var` in Node🔥 👀 🔥
`config` is already `this.config` in the Notifications class and it is set after you call the constructor for the superclass
`diff` could be converted to `this.diff`, probably?